modules/apps/analytics/analytics-client-js/src/plugins/dxp.js

```diff
@@ -54,7 +59,41 @@ function dxp(analytics) {
 		});
 	}
 
+	/**
+	 * Checks based on the URL param if it is a configuration portlet
+	 */
+	function isConfigurationPortlet() {
+		const searchParams = getSearchParams();
+
+		const portletId = searchParams.get(PARAM_PORTLET_ID_KEY);
+
+		if (portletId === PARAM_CONFIGURATION_PORTLET_NAME) {
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * Checks based on the URL param if the page is in view mode
+	 */
+	function isViewMode() {
+		const searchParams = getSearchParams();
+
+		const mode = searchParams.get(PARAM_MODE_KEY) || PARAM_VIEW_MODE;
+
+		if (mode === PARAM_VIEW_MODE) {
+			return true;
+		}
+
+		return false;
+	}
+
 	if (window.Liferay && window.Liferay.SPA) {
+		if (isConfigurationPortlet() || !isViewMode()) {
```

Comment:
`getSearchParams()` is called twice, but I guess this shouldn't be a noticeable performance hit.